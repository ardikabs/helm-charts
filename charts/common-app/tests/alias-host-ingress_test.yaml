suite: enable alias host ingress test
templates:
  - "ingress.yaml"
values:
  - "values/alias-host-ingress.yaml"
release:
  name: podinfo
  namespace: testing
tests:
  - it: should create an Ingress
    template: ingress.yaml
    asserts:
      - isKind:
          of: Ingress
      - hasDocuments:
          count: 3
      - matchSnapshot:
          path: spec
