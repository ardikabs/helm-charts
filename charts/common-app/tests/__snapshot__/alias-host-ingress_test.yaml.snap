should create an Ingress:
  1: |
    ingressClassName: nginx
    rules:
      - host: podinfo.ardikabs.com
        http:
          paths:
            - backend:
                service:
                  name: podinfo
                  port:
                    name: http
              path: /
              pathType: Prefix
    tls:
      - hosts:
          - podinfo.ardikabs.com
        secretName: podinfo-ardikabs-com-tls
  2: |
    ingressClassName: nginx
    rules:
      - host: alias1.ardikabs.com
        http:
          paths:
            - backend:
                service:
                  name: podinfo
                  port:
                    name: http
              path: /
              pathType: Prefix
    tls:
      - hosts:
          - alias1.ardikabs.com
        secretName: alias-ardikabs-com-tls
  3: |
    ingressClassName: nginx
    rules:
      - host: alias2.ardikabs.com
        http:
          paths:
            - backend:
                service:
                  name: podinfo
                  port:
                    name: http
              path: /
              pathType: Prefix
    tls:
      - hosts:
          - alias2.ardikabs.com
        secretName: alias2-ardikabs-com-tls
